include/frame.h

```diff
@@ -251,23 +262,23 @@ class Frame : public RefCounted {
               ords.push_back(ord);
             }
           }
-          units[player_id][i] = unit;
+          units[player_id][i]        = unit;
           units[player_id][i].orders = std::move(ords);
         }
       }
       // For resources: keep the ones of the next frame
       if (next_frame.resources.find(player_id) != next_frame.resources.end()) {
-        auto next_res = next_frame.resources.at(player_id);
-        resources[player_id].ore = next_res.ore;
-        resources[player_id].gas = next_res.gas;
-        resources[player_id].used_psi = next_res.used_psi;
+        auto next_res                  = next_frame.resources.at(player_id);
+        resources[player_id].ore       = next_res.ore;
+        resources[player_id].gas       = next_res.gas;
+        resources[player_id].used_psi  = next_res.used_psi;
         resources[player_id].total_psi = next_res.total_psi;
```

Comment:
Uh, I think this is quite ugly actually (and it will change cause a lot of re-formatting). I'll try to look for a more appropriate config option, but if there's nothing else we can use `// clang-format on` and `// clang-format off`.